### DISCUSSION TITLE
Eigenes Modell anlernen

### DISCUSSION THREAD

**Discussion by ThomasKlesse** [jomjol/AI-on-the-edge-device#2683] [General]
Hallo,

großes Lob an den Entwickler. Mein Gaszähler liefert mit dem AI on edge device seit über einen Jahr zuverlässig seine Stände.
Beim Wasserzähler klappt es leider nicht so gut. Der Grund liegt in der Anzeige. Die weißen Ziffern auf schwarzem Grund werden 100% erkannt, bei den weißen Ziffern auf rotem Grund passt es nur sehr selten. 

Ich wollte nun mein eigenes Modell trainieren. Die Bilder werden mit der letzten FW auch einfach auf der SD gespeichert.

Dabei ist mir aufgefallen, dass die Anleitung nicht mehr aktuell sein kann: 
https://jomjol.github.io/AI-on-the-edge-device-docs/Learn-models-with-your-own-images/

die Beschreibung passt nicht zu den Dateinamen im Repo ...
und in den Repositories liegen nicht die aktuellen tflite-Modelle. Zumindest ist im aktuellen FW-Ordner eine andere Version als im GIT.
Es kann auch sein, dass ich durch die Fülle an Infos etwas übersehen habe. Leider läuft das Jupyter-Script aus dem GIT auch nicht durch.

Meine Frage deshalb, wie und mit welchem Script kann ich für die digitalen Zahlen (für cont, Zähler) ein Modell  mit meinen Daten anlernen?

Vielen Dank
Thomas

**Comment by caco3** [comment]
If you see issues in the docs we would be happy to get some help fixing it. Just create a pull request in https://github.com/jomjol/AI-on-the-edge-device-docs/tree/main/docs

@haverland FYI
